scripts/engine/world.py

```diff
@@ -259,13 +262,13 @@ def create_fov_map() -> tcod.map.Map:
     width = gamemap.width
     height = gamemap.height
 
-    fov_map = tcod.map_new(width, height)
+    fov_map = np.zeros((width, height), dtype=bool, order="F")
```

Comment:
Why the move to numpy? And what is `np.zeros`?

Reply:
Because tcod.map_new is deprecated and the library recommends using `numpy` from now on. Always is necessary in order to combine the FOV maps. In this case `np.zeros` creates a matrix of size `(width, height)` (basically a grid) where each value is a bool with `False `as default.